recipe/test_script_exist.bat

```diff
@@ -0,0 +1,21 @@
+@echo off
+REM As a workaround for GDAL apps unexpectedly returning error codes, this
```

Comment:
Note: GDAL 3.8.0RC2 has been issued with the fix for that issue

Reply:
@rouault, is there also a plan to move from scripts to entry points to better support Windows?

Reply:
@xylar I'm waiting for feedback. See https://github.com/conda-forge/gdal-feedstock/issues/834#issuecomment-1798334805

Reply:
Ah, sorry, I had noticed that comment and just forgot.  Thanks!